### PR TITLE
Auto populate script

### PR DESCRIPTION
**Pull request by ajparsons**
This PR adjusts data loading scripts to be quicker for loading initial sets of data, and creates a new `script/quick-populate` to make loading a minimal set of data consisutent.

This replaces https://github.com/mysociety/theyworkforyou/pull/1591 because that was merging into an outdated version of a branch that has now been merged into master.

I have adjusted the populate scripts based on the feedback in that PR request.

### CODE REVIEW

File: scripts/json2db.pl [hunk 1]
```diff
@@ -82,9 +99,13 @@
 
 sub process_motions {
     my ($policy, $dreamid) = @_;
+    $start_transaction->execute();
```
Comment (dracos) [inline scripts/json2db.pl:102]: Bit confused why this file calls transaction SQL, whereas another one you've changed you've altered the AutoCommit value. Best to do both the same way, unless there's a reason?

File: scripts/json2db.pl [hunk 2]
```diff
@@ -13,6 +13,14 @@
 
 my $parldata = mySociety::Config::get('RAWDATA');
 
+my $verbose = 0;
+for( @ARGV ){
+  if( $_ eq "--verbose" ){
```
Comment (dracos) [inline scripts/json2db.pl:18]: Could spacing generally match the existing spacing in the same file, just for consistency :)

**Review by dracos — CHANGES_REQUESTED** [review]
Only real query is about the two different methods of transactions, probably misunderstood something.

File: scripts/json2db.pl (hunk 1 above)
Comment (ajparsons) [reply scripts/json2db.pl]: I've switched to this to the AutoCommit approach used elsewhere.  Run the import and it seems fine.

File: scripts/json2db.pl (hunk 2 above)
Comment (ajparsons) [reply scripts/json2db.pl]: Amended this spacing.

**Review by dracos — APPROVED** [review]
I meant the spaces round the brackets as well, but is fine ;)

**Pull request merged**